sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -477,8 +477,11 @@ def expired(self):
 
         :rtype: bool
         """
-        if self._receiver._session_id:  # pylint: disable=protected-access
-            raise TypeError("Session messages do not expire. Please use the Session expiry instead.")
+        try:
+            if self._receiver._session_id:  # pylint: disable=protected-access
```

Comment:
let's align the session check using attribute self._receiver.session instead of self._receiver._session_id?